.github/workflows/ruby.yml

```diff
@@ -13,29 +13,35 @@ jobs:
     name: test (ruby ${{ matrix.ruby }} / rails ${{ matrix.rails_version }} / blacklight ${{ matrix.blacklight_version }} ${{ matrix.additional_name }})
     strategy:
       matrix:
-        ruby: ["3.2", "3.3", "3.4"]
+        ruby: ["3.4"]
         rails_version: ["8.0.1"]
         blacklight_version: ["~> 8.0"]
         experimental: [false]
-        additional_engine_cart_rails_options: [""]
-        additional_name: [""]
+        additional_engine_cart_rails_options: ["--css bootstrap"]
+        additional_name: ["/ importmap"]
```

Comment:
I believe this may be confusing, as don't all of the builds other than the "/ sprockets" one use importmap? Can we just omit this?

Reply:
I can omit this, I am not attached. I was following what I saw in blacklight_range_limit.

Reply:
done!